### DISCUSSION TITLE
Create a Firebird Dataset

### DISCUSSION THREAD

**Discussion by fabiools** [getredash/redash#6390] [Q&A]
Hi guys,

First off all thank you so much for effort applyed in this project that is amazing! I'm trying to connect to firebird but unsucessful. Its possible do this natively?

In case of not possible, have any docs about how to create a new Dataset? I want to contribute adding this into the project!

One more time, thank so much and thanks for all help!

Regards from Brazil!

**Comment by justinclift** [comment]
We don't yet have Firebird as one of the available "Data Source" options.  It's probably not super difficult to add, but it would need someone familiar with Python and (a small amount of) Javascript experience to pick up the task.

Is that your kind of thing? :smile:
